config/manager/manager.yaml

```diff
@@ -50,7 +50,7 @@ spec:
             - "--logtostderr"
             - "--exempt-namespace=gatekeeper-system"
             - "--operation=webhook"
-          image: openpolicyagent/gatekeeper:v3.6.0
+          image: openpolicyagent/gatekeeper:v3.7.0-beta.1
```

Comment:
updating tags to `v3.7.0-beta.1` as this was missed when we cut the last release.